keyboards/hazel/chaz/rp2040ce/info.json

```diff
@@ -0,0 +1,65 @@
+{
+    "manufacturer": "Hazel",
+    "keyboard_name": "Chaz",
+    "maintainer": "jasonhazel",
+    "url": "https://github.com/hazels-garage/chaz",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "nkro": false
+    },
+    "matrix_pins": {
+        "cols": ["GP9", "GP8", "GP7", "GP6", "GP5"],
+        "rows": ["GP0", "GP2", "GP4", "GP28", "GP1", "GP3", "GP27", "GP29"]
+    },
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "usb": {
+        "vid": "0x4A48",
+        "pid": "0x435A",
+        "device_version": "1.0.0"
+    },
+    "layouts": {
```

Comment:
the layouts specified in `promicro` and `rp2040ce` are identical. Could you move "layouts" to `info.json` in directory above (keyboard level)? if done previously, please remove.